i18n/messages/nl.json

```
@@ -804,6 +824,11 @@
     "description": "Tooltip for 'Multiple values' builder field",
     "originalDefault": "Are there multiple values possible for this field?"
   },
+  "WXpj4j": {
+    "defaultMessage": "Gebruikers kunnen een punt aanduiden op de kaart",
```

```suggestion
    "defaultMessage": "Gebruikers kunnen een punt aanduiden op de kaart.",
```
ironic that the "punt" sentence is missing a "punt" :laughing: